src/sentry/search/events/fields.py

```diff
@@ -538,6 +546,7 @@ def resolve_orderby(orderby, fields, aggregations):
     those that are currently selected.
     """
     orderby = orderby if isinstance(orderby, (list, tuple)) else [orderby]
+    equation_aliases = [equation[-1] for equation in equations] if equations is not None else []
```

Comment:
any particular reason this a list rather than just a single item?

Reply:
:+1: from slack convo, this was from confusion around the loop included here. Split it up for future readability